### ISSUE TITLE
systemd service not found

### ISSUE THREAD

**Issue by vadi2** [matanui159/ReplaySorcery#2]
On Ubuntu 20.04 LTS:

```
 vadi   master  ~  Programs  ReplaySorcery  sudo make -C bin install
[sudo] password for vadi: 
make: Entering directory '/home/vadi/Programs/ReplaySorcery/bin'
make[1]: Entering directory '/home/vadi/Programs/ReplaySorcery/bin'
make[2]: Entering directory '/home/vadi/Programs/ReplaySorcery/bin'
make[2]: Leaving directory '/home/vadi/Programs/ReplaySorcery/bin'
[ 20%] Built target x264
make[2]: Entering directory '/home/vadi/Programs/ReplaySorcery/bin'
make[2]: Leaving directory '/home/vadi/Programs/ReplaySorcery/bin'
[ 40%] Built target jpeg
make[2]: Entering directory '/home/vadi/Programs/ReplaySorcery/bin'
make[2]: Leaving directory '/home/vadi/Programs/ReplaySorcery/bin'
[ 60%] Built target backtrace
make[2]: Entering directory '/home/vadi/Programs/ReplaySorcery/bin'
make[2]: Leaving directory '/home/vadi/Programs/ReplaySorcery/bin'
[100%] Built target replay-sorcery
make[1]: Leaving directory '/home/vadi/Programs/ReplaySorcery/bin'
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/bin/replay-sorcery
-- Installing: /usr/local/OFF/replay-sorcery.service
make: Leaving directory '/home/vadi/Programs/ReplaySorcery/bin'
 vadi   master  ~  Programs  ReplaySorcery  systemctl --user enable --now replay-sorcery
Failed to enable unit: Unit file replay-sorcery.service does not exist.
 vadi   master  ~  Programs  ReplaySorcery  

```

**Comment by bennetthardwick** [comment]
ReplaySorcery installs to `/usr/lib/systemd/user`. Do you have other services in there that work?

**Comment by matanui159** [comment]
Its installing to `/usr/local/OFF` maybe I broke the cmake file

**Comment by matanui159** [comment]
Turns out Cmake only does bools with `option` and you have to do some weird `set(<name> CACHE STRING)` to get it working with strings :facepalm: 

Kinda wish meson has support for external projects its alot nicer.

**Comment by matanui159** [comment]
Fixed in b70061f
Sorry about that :p
